frontend/app/model/FeedbackType.scala

```diff
@@ -0,0 +1,38 @@
+package model
+
+import configuration.Email._
+
+import scala.collection.immutable.Seq
+
+trait FeedbackType {
+  val description: String
+  val email: String = membershipSupport
+  def slug: String = description.filter(_.isLetterOrDigit).toLowerCase
+}
+
+object FeedbackType {
+  val all: Seq[FeedbackType] = List(CustomerService, AccessibilityIssue, MembershipFeedback, GuardianLive, Journalism)
+  def fromSlug(slug: String) = all.find(_.slug == slug)
+}
```

Comment:
Seeing all this makes me think we should try https://github.com/lloydmeta/enumeratum - maybe in a different PR